File: ci cislo je delitelne zadanym cislom/treti.cpp
#include <stdio.h>
#include  <math.h>

int main(){
	
	int a, b, zvysok;
	
	printf("Zadaj cislo:  ");
	scanf("%d",&a);
	
	printf("Zadaj cislo ktorym chces delit predchadzajuce cislo: ");
	scanf("%d",&b);
	
	zvysok = a % b;
	
	if(zvysok == 0){
		printf("Cislo %d je delitelne cislom %d", a,b);
	}else{
		printf("Cislo %d nie je delitelne cislom %d", a,b);
	}
	
}

File: ci cislo je delitelne zadanym cislom/treti.py
import math

a = int(input("Zadaj cislo: "))
b = int(input("Zadaj cislo ktorým chceš deliť predchádzajúce číslo: "))

zvysok = a % b;

if(zvysok == 0):
    print("Číslo ", a, "je deliteľné číslom ", b)
else:
    print("Číslo ", a, "nie je deliteľné číslom ", b)
